packages/discord.js/src/structures/CommandInteraction.js

```diff
@@ -45,6 +45,18 @@ class CommandInteraction extends BaseInteraction {
      */
     this.commandGuildId = data.data.guild_id ?? null;
 
+    /**
+     * Mapping of installation contexts that the interaction was authorized for the related user or guild ids
+     * @type {APIAuthorizingIntegrationOwnersMap}
```

Comment:
Should be added to util/APITypes.js

Also, I think this property should link to the docs, i.e. `@see {@link https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-authorizing-integration-owners-object}`

Reply:
Isn't the link to the d-types enough?